api/utils/extapi.go

```diff
@@ -0,0 +1,35 @@
+package utils
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+func FetchInfo(idParam int) string {
+
+	id := strconv.Itoa(idParam)
+
+	resp, err := http.Get("https://rupaulsdragrace.fandom.com/api.php/?action=query&prop=extracts&exlimit=1&explaintext=true&pageids=" + id + "&format=json")
```

Comment:
I know this 2nd deliverable is still somehow WIP, but remember to not hardcode the http client and the host you're trying to reach